library/core/resource-loader/src/main/java/org/quiltmc/qsl/resource/loader/api/package-info.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * The Resource Loader.
+ *
+ * <p><h3>Quick note about vocabulary in Resource Loader and Minecraft:</h3>
+ * <ul>
+ *  <li>Resource Pack refers to both client-sided resource pack and data pack.</li>
+ *  <li>Virtual Resource Pack refers to a resource pack that may be generated at runtime, or simply doesn't exist directly on disk.</li>
+ *  <li>Group Resource Pack refers to a virtual resource pack that groups multiple resource packs together.</li>
+ * </ul>
+ * </p>
+ *
+ * <p><h3>Modded Resource Pack Handling</h3></p>
+ * <p>The Resource Loader will create a resource pack for each mod that provides resources in {@code assets} or {@code data}
+ * sub-directories.
+ * Those mod resource packs are grouped into the default resource pack, they can override the default resource pack but unrecommended.</p>
```

Comment:
```suggestion
 * Those mod resource packs are grouped into the default resource pack. They can override the default resource pack, but it is not recommended.</p>
```
Also, why is it not recommended? This seems like it could use a "because".